src/pages/physical-count-itemDetails/index.js

```diff
@@ -0,0 +1,616 @@
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { useContext, useEffect, useState } from 'react'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { Grid } from '@mui/material'
+import { ControlContext } from 'src/providers/ControlContext'
+import { useResourceQuery } from 'src/hooks/resource'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { SCRepository } from 'src/repositories/SCRepository'
+import FormShell from 'src/components/Shared/FormShell'
+import { useForm } from 'src/hooks/form'
+import * as yup from 'yup'
+import { DataGrid } from 'src/components/Shared/DataGrid'
+import { InventoryRepository } from 'src/repositories/InventoryRepository'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { SystemChecks } from 'src/resources/SystemChecks'
+import toast from 'react-hot-toast'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { getFormattedNumber } from 'src/lib/numberField-helper'
+
+const PhysicalCountItemDe = () => {
+  const { getRequest, postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+  const [siteStore, setSiteStore] = useState([])
+  const [controllerStore, setControllerStore] = useState([])
+  const [filteredItems, setFilteredItems] = useState([])
+  const [editMode, setEditMode] = useState(false)
+  const [disSkuLookup, setDisSkuLookup] = useState(false) //better to put them in formik not states if possible
+  const [jumpToNextLine, setJumpToNextLine] = useState(false) //better to put them in formik not states if possible
+  const [showDefaultQty, setShowDefaultQty] = useState(false) //better to put them in formik not states if possible
+  const [disableItemDuplicate, setDisableItemDuplicate] = useState(false) //better to put them in formik not states if possible
+
```

Comment:
please remove these comments, if needed for someone, send it to them privately

Reply:
we need the comments for future work of the task IN THIS SAME SCREEN 
it will be cleared later on